terraform/modules/iam-common/hostegraphite.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_iam_policy" "hostedgraphite" {
+  name = "Hostedgraphite"
+  policy = <<EOF
+{
+    "Statement": [
+        {
+            "Sid": "PermissionsForMetrics",
+            "Effect": "Allow",
+            "Action": [
```

Comment:
Let's keep cloudwatch and ec2 permissions, everything else we don't have or wouldn't want in Grafana anyway.